distribution/lib/Standard/Snowflake/0.0.0-dev/src/Snowflake_Details.enso

```diff
@@ -0,0 +1,58 @@
+from Standard.Base import all
+import Standard.Base.Data.Numbers.Number_Parse_Error
+import Standard.Base.Errors.Illegal_State.Illegal_State
+
+import Standard.Database.Connection.Connection_Options.Connection_Options
+import Standard.Database.Connection.Credentials.Credentials
+
+import project.Internal.Snowflake_Connection.Snowflake_Connection
+
+polyglot java import net.snowflake.client.jdbc.SnowflakeDriver
+
+type Snowflake_Details
+    ## Connect to a Snowflake database.
+
+       Arguments:
+       - account: The name of the account to connect to.
+       - credentials: The credentials to use for the connection.
+       - database: The name of the database to connect to.
+       - schema: The name of the schema to connect to.
+       - warehouse: The name of the warehouse to use.
+    Snowflake account:Text credentials:Credentials database:Text="SNOWFLAKE" schema:Text="PUBLIC" warehouse:Text=""
+
+    ## PRIVATE
```

Comment:
I'd suggest to move these `## PRIVATE` methods to another `private` module and make them _extension methods_.

Reply:
The methods on the details class make the Database.connect method work so need to be attached to the type.